plone/recipe/haproxy/__init__.py

```diff
@@ -30,7 +26,7 @@ def __init__(self, buildout, name, options):
         self.download_cache = buildout['buildout'].get('download-cache')
         self.install_from_cache = buildout['buildout'].get(
             'install-from-cache')
-        if isinstance(self.install_from_cache, (str, unicode)):
+        if isinstance(self.install_from_cache, (str, bytes)):
             self.install_from_cache = self.install_from_cache.lower() == 'true'
```

Comment:
These two lines look fishy too me:
```
if isinstance(self.install_from_cache, (str, bytes)):
    self.install_from_cache = self.install_from_cache.lower() == 'true'
```
because on Python3 bytes can never be equal to a text:
```
>>> b"True".lower() == "true"
False
```
A fix might be:
```
self.install_from_cache = self.install_from_cache.lower() in ('true', b'true')
```
or to be sure `self.install_from_cache` is always a native string.

Reply:
@ale-rt  you're right ... I could not re-open this PR so I've made a new one with the fix: see https://github.com/plone/plone.recipe.haproxy/pull/4